theano/compile/function_module.py

```diff
@@ -751,7 +752,13 @@ def checkSV(sv_ori, sv_rpl):
         f_cpy.maker.fgraph.name = name
         return f_cpy
 
-    def __call__(self, *args, **kwargs):
+    def apply(self, inputs, outputs):
+        # TODO: assert that there's a list of outputs
+        self.output_idx = outputs # indirectly passing output indices
+        # assuming for now I pass all the inputs
+        return [self.__call__(*inputs)[i] for i in outputs]
+
+    def __call__(self, *args, **kwargs): # sygi: what those kwargs mean?
```

Comment:
**something

in a function signature, cause something to be a dict with all keyword parameter that aren't matching previous parameter.

Currently, there is mostly just one useful that exist and it exist only for the CVM, it is n_calls=N. If passed, this will do a loop in C where the function will be executed N times. I think we return only the last result. This is useful for small function to help lower the overhead when the function don't take any inputs.